Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/pocketwalk'...
Compiling './lib/python3.5/site-packages/pocketwalk/core.py'...
***   File "./lib/python3.5/site-packages/pocketwalk/core.py", line 34
    print(f"Got exception {type(error)}: {error}... trying again...")
                                                                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pocketwalk/plugger.py'...
***   File "./lib/python3.5/site-packages/pocketwalk/plugger.py", line 36
    namespace = f'{self._namespace.lower()}_{target.__name__.lower()}'
                                                                     ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pocketwalk/plugins'...
Compiling './lib/python3.5/site-packages/pocketwalk/plugins/config.py'...
***   File "./lib/python3.5/site-packages/pocketwalk/plugins/config.py", line 51
    config_dict[f'{tool}_targets'] = self._glob_paths(config_dict[f'{tool}_targets'])
                                ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pocketwalk/plugins/context_manager.py'...
***   File "./lib/python3.5/site-packages/pocketwalk/plugins/context_manager.py", line 95
    target_files = config[f'{this_tool}_targets']
                                               ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pocketwalk/plugins/tool_runner.py'...
***   File "./lib/python3.5/site-packages/pocketwalk/plugins/tool_runner.py", line 78
    print(f"Starting tools: {tools_to_start}")
                                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pocketwalk/plugins/vcs.py'...
***   File "./lib/python3.5/site-packages/pocketwalk/plugins/vcs.py", line 85
    tracked += config[f'{this_tool}_targets']
                                           ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pocketwalk/shell'...
Listing './lib/python3.5/site-packages/pocketwalk-0.2.2.dist-info'...
Listing './lib/python3.5/site-packages/pytoml'...
Listing './lib/python3.5/site-packages/pytoml-0.1.14.egg-info'...
Listing './lib/python3.5/site-packages/runaway'...
Compiling './lib/python3.5/site-packages/runaway/core.py'...
***   File "./lib/python3.5/site-packages/runaway/core.py", line 56
    message = f"\naction: {action}\nqueue:\n"
                                            ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/runaway/extras'...
Compiling './lib/python3.5/site-packages/runaway/signals.py'...
***   File "./lib/python3.5/site-packages/runaway/signals.py", line 166
    raise ValueError(f"Cannot wait for minimum_done of {minimum_done} with only {len(futures)} futures supplied.")
                                                                                                                ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/runaway/testing.py'...
***   File "./lib/python3.5/site-packages/runaway/testing.py", line 12
    raise AssertionError(f'{first} != {second}')
                                              ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/runaway-0.2.1.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/utaw'...
Listing './lib/python3.5/site-packages/utaw-0.2.0.dist-info'...
Listing './lib/python3.5/site-packages/wrapt'...
Listing './lib/python3.5/site-packages/wrapt-1.10.11.egg-info'...
Listing './share'...
Listing './share/python-wheels'...
